2020-10-29 05:41:24.753 - [Info] MITM Version: 1.26
2020-10-29 05:41:24.759 - [Info] Auto Access Enabled: true
2020-10-29 05:41:24.760 - [Debug] [Init] Auto Access Barrier: true
2020-10-29 05:41:24.783 - [Info] SSH man-in-the-middle server for 172.20.0.202 listening on 0.0.0.0:10001
2020-10-29 05:43:57.458 - [Debug] [Connection] Attacker connected: 193.105.134.45 | Client Identification: SSH-2.0-OpenSSH_5.2
2020-10-29 05:43:58.989 - [Debug] [Auth] Attacker 193.105.134.45 trying to authenticate with "none"
2020-10-29 05:43:58.990 - [Debug] [Auth] Attacker authentication error: No authentication method provided
2020-10-29 05:43:59.278 - [Debug] [Auth] Attacker 193.105.134.45 trying to authenticate with "password"
2020-10-29 05:43:59.281 - [Debug] [Auto Access] Attacker: 193.105.134.45, Threshold: 1, Attempts: 1
2020-10-29 05:43:59.281 - [Debug] [Auto Access] Compromising the honeypot
2020-10-29 05:43:59.281 - [Debug] [Auto Access] Adding the following credentials: "22:master"
2020-10-29 05:43:59.951 - [Debug] [Auth] Valid credentials - Password Authentication
2020-10-29 05:43:59.951 - [Debug] [LXC] Attempting to connect to the honeypot: 172.20.0.202
2020-10-29 05:44:15.342 - [Debug] [LXC-Auth] Attacker authenticated and is inside container
2020-10-29 05:44:16.465 - [Debug] [Connection] Attacker closed connection
2020-10-29 05:44:17.021 - [Debug] [LXC] Container's OpenSSH server ended connection
2020-10-29 05:44:17.022 - [Debug] [LXC] Container's OpenSSH server closed connection
2020-10-29 05:44:22.443 - [Debug] [Connection] Attacker connected: 195.3.147.47 | Client Identification: SSH-2.0-paramiko_2.0.2
2020-10-29 05:44:23.929 - [Debug] [Auth] Attacker 195.3.147.47 trying to authenticate with "none"
2020-10-29 05:44:23.929 - [Debug] [Auth] Attacker authentication error: No authentication method provided
2020-10-29 05:44:24.199 - [Debug] [Auth] Attacker 195.3.147.47 trying to authenticate with "password"
2020-10-29 05:44:24.200 - [Debug] [Auth] Attacker authentication error: Invalid credentials - User does not exist
2020-10-29 05:44:24.202 - [Debug] [Auth] Attacker: 195.3.147.47 has so far made 1 attempts. Remaining: 5 attempts
2020-10-29 05:44:24.470 - [Debug] [Connection] Attacker closed the connection
2020-10-29 05:45:40.806 - [Debug] [Connection] Attacker connected: 109.236.91.85 | Client Identification: SSH-2.0-JSCH_0.1.51
2020-10-29 05:45:41.939 - [Debug] [Auth] Attacker 109.236.91.85 trying to authenticate with "none"
2020-10-29 05:45:41.940 - [Debug] [Auth] Attacker authentication error: No authentication method provided
2020-10-29 05:45:42.157 - [Debug] [Auth] Attacker 109.236.91.85 trying to authenticate with "password"
2020-10-29 05:45:42.158 - [Debug] [Auth] Attacker authentication error: Invalid credentials - User does not exist
2020-10-29 05:45:42.159 - [Debug] [Auth] Attacker: 109.236.91.85 has so far made 1 attempts. Remaining: 5 attempts
2020-10-29 05:45:42.379 - [Debug] [Connection] Attacker closed the connection
2020-10-29 05:45:58.976 - [Debug] [Connection] Attacker connected: 34.65.109.41 | Client Identification: SSH-2.0-libssh2_1.4.3
2020-10-29 05:46:00.390 - [Debug] [Auth] Attacker 34.65.109.41 trying to authenticate with "password"
2020-10-29 05:46:00.391 - [Debug] [Auth] Attacker authentication error: Invalid credentials - Password Authentication Failure
2020-10-29 05:46:00.392 - [Debug] [Auth] Attacker: 34.65.109.41 has so far made 1 attempts. Remaining: 5 attempts
2020-10-29 05:46:00.637 - [Debug] [Connection] Attacker closed the connection
2020-10-29 05:46:17.643 - [Debug] [Connection] Attacker connected: 34.65.109.41 | Client Identification: SSH-2.0-libssh2_1.4.3
2020-10-29 05:46:18.728 - [Debug] [Auth] Attacker 34.65.109.41 trying to authenticate with "password"
2020-10-29 05:46:18.730 - [Debug] [Auth] Attacker authentication error: Invalid credentials - Password Authentication Failure
2020-10-29 05:46:18.731 - [Debug] [Auth] Attacker: 34.65.109.41 has so far made 1 attempts. Remaining: 5 attempts
2020-10-29 05:46:18.973 - [Debug] [Connection] Attacker closed the connection
2020-10-29 05:46:36.211 - [Debug] [Connection] Attacker connected: 34.65.109.41 | Client Identification: SSH-2.0-libssh2_1.4.3
2020-10-29 05:46:37.301 - [Debug] [Auth] Attacker 34.65.109.41 trying to authenticate with "password"
2020-10-29 05:46:37.302 - [Debug] [Auth] Attacker authentication error: Invalid credentials - Password Authentication Failure
2020-10-29 05:46:37.303 - [Debug] [Auth] Attacker: 34.65.109.41 has so far made 1 attempts. Remaining: 5 attempts
2020-10-29 05:46:37.541 - [Debug] [Connection] Attacker closed the connection
2020-10-29 05:46:54.413 - [Debug] [Connection] Attacker connected: 34.65.109.41 | Client Identification: SSH-2.0-libssh2_1.4.3
2020-10-29 05:46:55.435 - [Debug] [Connection] Attacker connected: 195.3.147.47 | Client Identification: SSH-2.0-paramiko_1.8.1
2020-10-29 05:46:56.629 - [Debug] [Auth] Attacker 34.65.109.41 trying to authenticate with "password"
2020-10-29 05:46:56.630 - [Debug] [Auth] Attacker authentication error: Invalid credentials - Password Authentication Failure
2020-10-29 05:46:56.631 - [Debug] [Auth] Attacker: 34.65.109.41 has so far made 1 attempts. Remaining: 5 attempts
2020-10-29 05:46:56.873 - [Debug] [Connection] Attacker closed the connection
2020-10-29 05:46:56.875 - [Debug] [Auth] Attacker 195.3.147.47 trying to authenticate with "none"
2020-10-29 05:46:56.875 - [Debug] [Auth] Attacker authentication error: No authentication method provided
2020-10-29 05:46:57.137 - [Debug] [Auth] Attacker 195.3.147.47 trying to authenticate with "password"
2020-10-29 05:46:57.138 - [Debug] [Auth] Attacker authentication error: Invalid credentials - User does not exist
2020-10-29 05:46:57.139 - [Debug] [Auth] Attacker: 195.3.147.47 has so far made 1 attempts. Remaining: 5 attempts
2020-10-29 05:46:57.397 - [Debug] [Connection] Attacker closed the connection
2020-10-29 05:47:12.219 - [Debug] [Connection] Attacker connected: 34.65.109.41 | Client Identification: SSH-2.0-libssh2_1.4.3
2020-10-29 05:47:13.765 - [Debug] [Auth] Attacker 34.65.109.41 trying to authenticate with "password"
2020-10-29 05:47:13.766 - [Debug] [Auth] Attacker authentication error: Invalid credentials - Password Authentication Failure
2020-10-29 05:47:13.768 - [Debug] [Auth] Attacker: 34.65.109.41 has so far made 1 attempts. Remaining: 5 attempts
2020-10-29 05:47:14.008 - [Debug] [Connection] Attacker closed the connection
2020-10-29 05:47:16.471 - [Debug] [Connection] Attacker connected: 195.3.147.47 | Client Identification: SSH-2.0-Nmap_SSH2_Hostkey  
2020-10-29 05:47:17.651 - [Debug] [Auth] Attacker 195.3.147.47 trying to authenticate with "none"
2020-10-29 05:47:17.651 - [Debug] [Auth] Attacker authentication error: No authentication method provided
2020-10-29 05:47:17.911 - [Debug] [Auth] Attacker 195.3.147.47 trying to authenticate with "password"
2020-10-29 05:47:17.912 - [Debug] [Auth] Attacker authentication error: Invalid credentials - User does not exist
2020-10-29 05:47:17.912 - [Debug] [Auth] Attacker: 195.3.147.47 has so far made 1 attempts. Remaining: 5 attempts
2020-10-29 05:47:18.172 - [Debug] [Connection] Attacker closed the connection
2020-10-29 05:47:29.602 - [Debug] [Connection] Attacker connected: 34.65.109.41 | Client Identification: SSH-2.0-libssh2_1.4.3
2020-10-29 05:47:30.690 - [Debug] [Auth] Attacker 34.65.109.41 trying to authenticate with "password"
2020-10-29 05:47:30.691 - [Debug] [Auth] Attacker authentication error: Invalid credentials - Password Authentication Failure
2020-10-29 05:47:30.692 - [Debug] [Auth] Attacker: 34.65.109.41 has so far made 1 attempts. Remaining: 5 attempts
2020-10-29 05:47:30.929 - [Debug] [Connection] Attacker closed the connection
2020-10-29 05:47:47.147 - [Debug] [Connection] Attacker connected: 34.65.109.41 | Client Identification: SSH-2.0-libssh2_1.4.3
2020-10-29 05:47:48.248 - [Debug] [Auth] Attacker 34.65.109.41 trying to authenticate with "password"
2020-10-29 05:47:48.249 - [Debug] [Auth] Attacker authentication error: Invalid credentials - Password Authentication Failure
2020-10-29 05:47:48.251 - [Debug] [Auth] Attacker: 34.65.109.41 has so far made 1 attempts. Remaining: 5 attempts
2020-10-29 05:47:48.491 - [Debug] [Connection] Attacker closed the connection
2020-10-29 05:48:04.333 - [Debug] [Connection] Attacker connected: 34.65.109.41 | Client Identification: SSH-2.0-libssh2_1.4.3
2020-10-29 05:48:05.818 - [Debug] [Auth] Attacker 34.65.109.41 trying to authenticate with "password"
2020-10-29 05:48:05.819 - [Debug] [Auth] Attacker authentication error: Invalid credentials - Password Authentication Failure
2020-10-29 05:48:05.820 - [Debug] [Auth] Attacker: 34.65.109.41 has so far made 1 attempts. Remaining: 5 attempts
2020-10-29 05:48:06.058 - [Debug] [Connection] Attacker closed the connection
2020-10-29 05:48:21.731 - [Debug] [Connection] Attacker connected: 34.65.109.41 | Client Identification: SSH-2.0-libssh2_1.4.3
2020-10-29 05:48:23.285 - [Debug] [Auth] Attacker 34.65.109.41 trying to authenticate with "password"
2020-10-29 05:48:23.287 - [Debug] [Auth] Attacker authentication error: Invalid credentials - Password Authentication Failure
2020-10-29 05:48:23.288 - [Debug] [Auth] Attacker: 34.65.109.41 has so far made 1 attempts. Remaining: 5 attempts
2020-10-29 05:48:23.534 - [Debug] [Connection] Attacker closed the connection
2020-10-29 05:48:38.867 - [Debug] [Connection] Attacker connected: 34.65.109.41 | Client Identification: SSH-2.0-libssh2_1.4.3
2020-10-29 05:48:40.341 - [Debug] [Auth] Attacker 34.65.109.41 trying to authenticate with "password"
2020-10-29 05:48:40.342 - [Debug] [Auth] Attacker authentication error: Invalid credentials - Password Authentication Failure
2020-10-29 05:48:40.343 - [Debug] [Auth] Attacker: 34.65.109.41 has so far made 1 attempts. Remaining: 5 attempts
2020-10-29 05:48:40.581 - [Debug] [Connection] Attacker closed the connection
2020-10-29 05:48:56.209 - [Debug] [Connection] Attacker connected: 34.65.109.41 | Client Identification: SSH-2.0-libssh2_1.4.3
2020-10-29 05:48:57.757 - [Debug] [Auth] Attacker 34.65.109.41 trying to authenticate with "password"
2020-10-29 05:48:57.759 - [Debug] [Auth] Attacker authentication error: Invalid credentials - Password Authentication Failure
2020-10-29 05:48:57.760 - [Debug] [Auth] Attacker: 34.65.109.41 has so far made 1 attempts. Remaining: 5 attempts
2020-10-29 05:48:58.004 - [Debug] [Connection] Attacker closed the connection
2020-10-29 05:49:13.619 - [Debug] [Connection] Attacker connected: 34.65.109.41 | Client Identification: SSH-2.0-libssh2_1.4.3
2020-10-29 05:49:15.407 - [Debug] [Auth] Attacker 34.65.109.41 trying to authenticate with "password"
2020-10-29 05:49:15.409 - [Debug] [Auth] Attacker authentication error: Invalid credentials - User does not exist
2020-10-29 05:49:15.410 - [Debug] [Auth] Attacker: 34.65.109.41 has so far made 1 attempts. Remaining: 5 attempts
2020-10-29 05:49:15.646 - [Debug] [Connection] Attacker closed the connection
